internal/agent/agent.go

```diff
@@ -101,9 +102,26 @@ func (a *Agent) Run(ctx context.Context) error {
 
 	executer := &executer.CommonExecuter{}
 
+	// TODO: expose through config
+	fsAlertThreshold := 10
+	fsWarnThreshold := 20
+	fsPaths := []string{"/"}
+	fsSyncDuration := time.Minute
+	fsTimeoutDuration := time.Second * 5
```

Comment:
I think these aren't used?

Reply:
They are passed NewFs to and consumed by NewFs via Manager. Eventually, this should have a config.